package.json

```diff
@@ -78,6 +78,7 @@
     "copy-dir": "0.3.0",
     "coveralls": "2.11.12",
     "deepcopy": "0.6.3",
+    "firefox-client": "0.3.0",
```

Comment:
Good catch. This was a flattened deps casualty (it's a sub dep defined in https://github.com/mozilla/node-firefox-connect/blob/master/package.json#L8)